src/ExternalModule.ts

```diff
@@ -110,11 +110,27 @@ export default class ExternalModule {
 						.map(name => `'${name}'`)
 						.join(', ')} and '${unused.slice(-1)}' are`;
 
+		const importersSet = new Set<string>()
+		for (const name of unused) {
+			const {importers, dynamicImporters} = this.declarations[name].module
+
+			if (Array.isArray(importers)) importers.forEach(v => importersSet.add(v))
+			if (Array.isArray(dynamicImporters)) dynamicImporters.forEach(v => importersSet.add(v))
```

Comment:
By the types, these checks should not be needed. This is also reflected in the coverage right now. Or do you see a scenario where there might be an issue?

Reply:
During my previous experiments there was a situation when `dynamicImporters` was `undefined`. I assume that the types are not quite correct. Therefore, for reinsurance, I added a check.